### PR TITLE
Added Microsoft.SyndicationFeed solution.

### PR DESCRIPTION
**Pull request by jimmyca15**

### CODE REVIEW

File: lab/src/Microsoft.SyndicationFeed/tests/Microsoft.SyndicationFeed.Tests.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <Folder Include="TestFeeds\">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Folder>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="TestFeeds\SimpleRssFeed.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
```
Comment (zhenlan) [inline lab/src/Microsoft.SyndicationFeed/tests/Microsoft.SyndicationFeed.Tests.csproj]: We can add SimpleRssFeed.xml when it is actually used by a test in the future.

Comment (jimmyca15) [inline lab/src/Microsoft.SyndicationFeed/tests/Microsoft.SyndicationFeed.Tests.csproj]: Agreed. I removed it.

**Comment by jimmyca15** [comment]
I will close and reopen this to refresh the cla requirement.

**Comment by jimmyca15** [comment]
The CLA did not update. I will submit an entirely new PR.

**Pull request closed without merging**